post-install.xql

```diff
@@ -40,22 +40,30 @@ declare function local:set-admin-password() as empty-sequence() {
         else ()
 };
 
+declare function local:first-run() as xs:boolean {
+    not(sm:list-groups() = 'mermedit')
+};
 
 declare function local:create-group() as empty-sequence() {
     sm:create-group('mermedit')
 };
 
 declare function local:change-group() as empty-sequence() {
```

Comment:
could we adjust this function name, or separate concerns? I think I'd prefer the latter and add a dedicated function e.g. `local:set-up-data-dir()` and add this to the set of "first-run" functions.

Reply:
I don't see why we should change this functions name. The purpose here is to determine if this is the first run. So using it in `if (local:first-run) then ` is what it is meant for. The method of determining that may be something to discuss. This was just the first thing that came to mind.

Reply:
Sorry, I did not anchor this comment correctly. My concern is about the function `local:change-group()` which is doing much more than what it's saying.